src/Form/Fields.php

```diff
@@ -268,7 +268,7 @@ public function reset(): static
 
 		// reset the values of each field
 		foreach ($this->data as $field) {
-			$field->fill(null);
+			$field->fillWithEmptyValue();
```

Comment:
`Suggestion:` Maybe instead of `$field->fillWithEmptyValue()` this should rather be `$field->reset()`.